docs/introduction.md

```diff
@@ -42,7 +44,36 @@ The final version of the previous execution state remains available through a le
 
 ## Flow
 
-Flow was designed to provide a blockchain that can scale while preserving composability in a single blockchain state. This is achieved by introducing several [node roles](#nodes), each with their own trade-off for the [scalability trilemma](https://aakash-111.medium.com/the-scalability-trilemma-in-blockchain-75fb57f646df).
+Most current blockchains are built as a homogenous system comprised of *full nodes*.
+Each full node is expected to collect and choose the transactions to be included in the next block, execute the block, come to a consensus over the output of the block with other full nodes, and finally sign the block and append it to the chain.
+
+Existing blockchain architectures have well documented throughput limitations. There are two main approaches to combat this:
+
+1. Layer-2 networks
+2. Sharding
+
+Layer-2 solutions include include networks like Bitcoin's Lightning and Ethereum's Plasma - these networks work off the main chain.
```

Comment:
```suggestion
Layer-2 solutions include include networks like Bitcoin's Lightning and Ethereum's Plasma — these networks work off the main chain.
```

Should be an _em dash_ (—) instead of an _en dash_ (-). Applies to the rest of the doc as well.